GiniSwitchSDK/GiniSwitchSDK/Classes/UIButton+ImageColor.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UIButton+ImageColor.swift
+//  GiniSwitchSDK
+//
+//  Created by Nikola Sobadjiev on 09.10.17.
+//
+
+import Foundation
+
+extension UIButton {
+    
+    var imageColor: UIColor {
```

Comment:
This only would work for `normal` state, right? So maybe you need to rename the property and extract the `set` functionality to a method?

Reply:
Since setting the color only for the normal state will generally work (it will go darker for highlighted and the button doesn't really get selected or disabled), I decided to leave this method but add an explicit explanation that it will only change that one state.

Additionally, I added a method for changing the other state colors.

Do you think it's enough @kikettas ?

Reply:
Más que suficiente! 👍